packages/zoltan2/src/algorithms/order/Zoltan2_AlgAMD.hpp

```diff
@@ -128,14 +128,15 @@ class AlgAMD : public Algorithm<Adapter>
     { }
 
     int globalOrder(
-      const RCP<GlobalOrderingSolution<typename Adapter::gno_t> > &solution) {
+      const RCP<GlobalOrderingSolution<typename Adapter::gno_t> > &/* solution */) {
         throw std::logic_error("AlgAMD does not yet support global ordering.");
     }
 
     int localOrder(
       const RCP<LocalOrderingSolution<typename Adapter::lno_t> > &solution)
     {
 #ifndef HAVE_ZOLTAN2_AMD
+      (void)solution;
```

Comment:
An odd line like this one should have a comment saying why it was added (e.g., "// needed to avoid unused-parameter warnings for function argument solution").  Otherwise, someone is likely to say someday, "That's weird" and remove it.

Reply:
Addressed in the latest commit.